event_links.rb

```diff
@@ -79,5 +79,13 @@ class Server < Sinatra::Base
     get /\/(c|cmp|championship)/i do
       redirect "https://www.firstchampionship.org"
     end
+    
+    # Redirects to the COVID information for the given event.
+    get /\/(e|event)\/(v|covid)\/([A-Za-z]+\d?)(\/(\d+))?/i do
```

Comment:
Any particular reason to use `v` instead of `c`? I'm pretty sure `c` isn't currently used.